pyke/kepfold.py

```diff
@@ -108,6 +108,19 @@ def kepfold(infile, period, bjd0, outfile=None, bindata=False,
 
     if outfile is None:
         outfile = infile.split('.')[0] + "-{}.fits".format(__all__[0])
+    # check if there is period or BJD0 information
+    if np.any([(period is None),(bjd0 is None)]):
```

Comment:
The only problem I see here is that this statement ignores the input from the user whenever only one of them (``period`` or ``bjd0``) is not ``None``.

Reply:
Roger, will change that